applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -82,19 +86,9 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
 }
 
 @displayNewslettersByTheme = {
-    @List(
-        "News roundups"  -> newsRoundUpEmails,
-        "News by topic"  -> newsEmails,
-        "Features"       -> featureEmails,
-        "Sport"          -> sportEmails,
-        "Culture"        -> cultureEmails,
-        "Lifestyle"      -> lifestyleEmails,
-        "Comment"        -> commentEmails,
-        "Work"           -> workEmails,
-        "From the papers" -> fromThePapersEmails
-    ).map { case (theme, newsletters) =>
+    @signupPage.metadata.groupedNewslettersResponses.map { case (theme, newsletters) =>
         <div class="newsletters-category" name="@theme">
-            @emailListCategoryList(theme, newsletters)
+        @emailListCategoryList(theme, newsletters)
```

Comment:
nitpick: I preferred the old indentation 🤷‍♂️

Reply:
I did too. I think IntelliJ got confused by the Twirl template. I'll change this back as part of a few other similar changes